signer/src/stacks/api.rs

```diff
@@ -634,7 +652,7 @@ pub struct GetNodeInfoResponse {
     /// The block header hash of the tip of the canonical stacks
     /// blockchain. This is hashed with the consensus hash to create the
     /// block id.
-    stacks_tip: BlockHeaderHash,
+    pub stacks_tip: BlockHeaderHash,
```

Comment:
Why the `pub` here? We generally don't want to use this as we only use block ids around (eg, `stacks_chain_tip`)